src/variational/VariationalInference.jl

```diff
@@ -248,76 +251,61 @@ end
 """
     vi(
```

Comment:
Since the interface is changing, could you target this PR to the `breaking` branch and add an entry to HISTORY.md explaining to users what has changed and how they need to adapt?